src/components/NavigasjonsTopp.tsx

```diff
@@ -1,60 +1,52 @@
 import React, { ReactElement } from "react";
-import { Link } from "react-router-dom";
-import styled, { css } from "styled-components";
-
-interface Lenke {
-  tittel: string;
-  url: string;
-  aktiv: boolean;
-}
-
-interface NavigasjonsToppProps {
-  lenker: Lenke[];
+import { useLocation, useNavigate } from "react-router-dom";
+import { Box, Heading, Tabs } from "@navikt/ds-react";
+import LinkAsTab from "@/components/LinkAsTab";
+import { fullNaisUrlDefault } from "@/utils/miljoUtil";
+import { MagnifyingGlassIcon } from "@navikt/aksel-icons";
+import {
+  enhetensMoterRoutePath,
+  mineMoterRoutePath,
+} from "@/routers/AppRouter";
+
+const texts = {
+  minOversikt: "Min oversikt",
+  enhetensOversikt: "Enhetens oversikt",
+  mineMoter: "Mine møter",
+  enhetensMoter: "Enhetens møter",
```

Comment:
Ble besluttet å beholde navnene "Mine møter" og "Enhetens møter" for å ikke endre for mye på en gang nå.